src/main/java/de/teamlapen/vampirism/entity/hunter/BasicHunterEntity.java

```diff
@@ -474,6 +477,16 @@ public void writeAdditional(CompoundNBT nbt) {
         }
     }
 
+    @Override
+    public void onDeath(DamageSource cause) {
+        if (cause.getTrueSource() instanceof PlayerEntity && this.villageAttributes == null) {
+            if (ItemStack.areItemStacksEqual(this.getItemStackFromSlot(EquipmentSlotType.HEAD), HunterVillageData.createBanner())) {
```

Comment:
Is it possible to use `#getBanner` or even a new method `isBanner` here to avoid creating a new item stack when one is cached anyway